src/components/shared/Sidebar/sidebar.styles.js

```diff
@@ -29,7 +30,11 @@ const fadeOut = keyframes`
 `;
 
 export const MenuItemsContainer = styled.div`
-  animation: ${({ isCollapsed }) => (isCollapsed ? fadeOut : fadeIn)}
+  animation: ${({ isCollapsed }) => {
+      if (isCollapsed === true) return fadeOut;
+      if (isCollapsed === false) return fadeIn;
+      return '';
+    }}
```

Comment:
can we revert this change here 
```suggestion
animation: ${({ isCollapsed }) => (isCollapsed ? fadeOut : fadeIn)}
```